src/GenericSpectator/Assets/Scripts/PlatformDeviceBootstrapper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;
+
+public class PlatformDeviceBootstrapper : MonoBehaviour
+{
+    /// <summary>
+    /// The device prefab for Windows platforms.
+    /// </summary>
+    [Tooltip("The device prefab for Windows platforms.")]
+    [SerializeField]
+    protected GameObject windowsDevicePrefab;
+
+    /// <summary>
+    /// The device prefab for the Android platform.
+    /// </summary>
+    [Tooltip("The device prefab for the Android platform.")]
+    [SerializeField]
+    protected GameObject androidDevicePrefab;
+
+    /// <summary>
+    /// The device prefab for the iOS platform.
+    /// </summary>
+    [Tooltip("The device prefab for the iOS platform.")]
+    [SerializeField]
+    protected GameObject iOSDevicePrefab;
+
+    protected void Awake()
+    {
+        GameObject devicePrfab;
```

Comment:
nit: devicePrefab

Reply:
Haha, I had to read that a couple times before I noticed... maybe I should up the font size on my monitor ;-).

Thanks for the catch!

---
In reply to: [337248663](https://github.com/microsoft/MixedReality-SpectatorView/pull/250#discussion_r337248663) [](ancestors = 337248663)